wporg-two-factor.php

```diff
@@ -28,21 +29,30 @@ function is_2fa_beta_tester() : bool {
 
 require_once __DIR__ . '/settings/settings.php';
 
+/*
+ * Make sure the WebAuthn plugin loads early, because all of our functions that call
+ * `Two_Factor_Core::is_user_using_two_factor()` etc assume that all providers are loaded. If WebAuthn is loaded
+ * too late, then `remove_capabilities_until_2fa_enabled()` would cause `get_enable_2fa_notice()` to be shown on
+ * the front end if WebAuthn is enabled and TOTP isn't.
+ */
+$webauthn = WebAuthn_Plugin::instance();
+$webauthn->init();
```

Comment:
```suggestion
$webauthn = WebAuthn_Plugin::instance();
$webauthn->init();
$webauthn->maybe_update_schema(); // This needs to run before plugins_loaded, as jetpack and wporg-two-factor do things way too early to the $current_user.
```

On WordPress.org this is also needed, as although it runs on `plugins_loaded`, Jetpack runs on `plugins_loaded` before it, which fetches the current user, which triggers `remove_super_admins_until_2fa_enabled()` before anything is ready to run.

Reply:
Thanks! merged 👍🏻